api/system/index.js

```diff
@@ -11,5 +11,7 @@ module.exports = function (options) {
   router.get('/swagger.yaml', systemController.swagger)
   router.get('/ping', systemController.ping)
   router.get('/ping', format)
+  router.get('/introspect', (req, res, next) => { systemController.introspect(req, res, next) })
```

Comment:
Quasiment identique à ```router.get('/introspect', systemController.introspect)```

Reply:
il y a un this dans la méthode donc pour pas le perdre, c'est obligé